org.mwc.debrief.core/src/org/mwc/debrief/core/ui/SWTEclipseHelper.java

```diff
@@ -54,15 +56,11 @@ public String askQuestion(final String title, final String question)
       @Override
       public void run()
       {
-        final String[] choices = new String[]
-        {"Cancel", "Alpha", "Bravo"};
-        final MessageDialog dialog = new MessageDialog(null, title, null,
-            question, MessageDialog.QUESTION, choices, 0); // cancel is the default
-        final int res = dialog.open();
-        if (res != 0)
-          answerVal.set(choices[res]);
-        else
-          answerVal.set(null);
+        InputDialog input = new InputDialog(null, title, question, "track name", null);
```

Comment:
pass `targetDisplay.getActiveShell()`

Reply:
we just chatted, and agreed to leave it as `null`